pkg/agent/variables.go

```diff
@@ -28,9 +29,28 @@ func getCustomDataVariables(config *datamodel.NodeBootstrappingConfiguration) pa
 			"reconcilePrivateHostsService": getBase64EncodedGzippedCustomScript(reconcilePrivateHostsService, config),
 			"configureAzure0Script":        getBase64EncodedGzippedCustomScript(kubernetesConfigAzure0Script, config),
 		},
+		// customData defined here is mainly used for Windows
+		"customData": paramsMap{
```

Comment:
I think it makes more sense to  have a separate getWindowsCustomDataVariables  - there is no much shared things in this method between windows and linux.